src/lib/auth/accounts/createMissingAccounts.ts

```diff
@@ -62,10 +58,23 @@ export async function createMissingAccounts(emails: string[], appId: string) {
       return {
         assignedExisting: assignExistingAccounts.count,
         createdNew: createNewAccounts.count,
-        assignedNew: assignNewAccounts.count
+        assignedNew: assignNewAccounts.count,
+        newAccounts: newAccounts
       };
     });
 
+    const capsuleAccounts = await createCapsuleAccounts(result.newAccounts.map(acc => ({ accountId: acc.id, email: acc.email })));
+
+    for (const capsuleAccount of capsuleAccounts) {
```

Comment:
I tihnk we can speed up those queries with sth like this:

```
async function updateUserAccounts(capsuleAccounts) {
  await prisma.$transaction(async (tx) => {
    const updates = capsuleAccounts.map(account => ({
      where: { email: account.email },
      data: {
        walletAddress: account.walletAddress,
        capsuleTokenVaultKey: account.capsuleTokenVaultKey
      }
    }));

    await tx.user.updateMany({
      data: updates
    });
  });
}
```
maybe you can quickly check it?